File: classmate/QuizApp/app/src/main/java/com/hikmatillo/quizapp/ResultActivity.java
package com.hikmatillo.quizapp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

import java.text.BreakIterator;

public class ResultActivity extends AppCompatActivity {
    private final TextView coins = new TextView(this);
    private final TextView trueAnswer = new TextView(this);
    private final TextView falseAnswer = new TextView(this);


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_result);

        String trueAns = getIntent().getStringExtra("TRUE_ANSWER");
        String falseAns = getIntent().getStringExtra("FALSE_ANSWER");
        String coin = getIntent().getStringExtra(Constant.COIN);


        loadView();
        loadDataToView(trueAns, falseAns, coin);

    }



    private void loadDataToView(String tA, String fA, String coin ) {

        trueAnswer.setText(tA);
        falseAnswer.setText(fA);
        coins.setText(coin);
    }

    private void loadView() {
        trueAnswer.findViewById(R.id.trueAnswer);
        falseAnswer.findViewById(R.id.falseAnswer);
        coins.findViewById(R.id.sadsc);
    }
}

File: classmate/WordGane/app/src/main/java/com/example/wordgane/ui/ResultActivity.kt
package com.example.wordgane.ui

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AlertDialog
import com.example.wordgane.R
import com.example.wordgane.core.BaseActivity
import com.example.wordgane.core.cashe.AppCashe
import com.example.wordgane.databinding.ActivityResultBinding

class ResultActivity : BaseActivity() {

    private lateinit var binding: ActivityResultBinding
    override fun setTheme() {
        when (AppCashe.getObject().getTheme()) {
            0 -> {
                binding.image.setBackgroundResource(R.drawable.back_photo2)
            }

            1 -> {
                binding.image.setBackgroundResource(R.drawable.back_photo3)
            }

            2 -> {
                binding.image.setBackgroundResource(R.drawable.back_photo4)
            }
        }

        val dialog = AlertDialog.Builder(this)
            .setTitle("Restart")
            .setIcon(R.drawable.baseline_restart_alt_24)
            .setMessage("Do you want to play again?")
            .setCancelable(false)
            .setPositiveButton("Yes") { _, _ -> startActivity(Intent(this, MainActivity::class.java) )}
            .setNegativeButton("No"){_,_ -> finish()}
            .show()
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityResultBinding.inflate(layoutInflater)
        setContentView(binding.root)



        loadView()

    }

    private fun loadView() {

        val coin = intent.getIntExtra("Coin", 0)

        binding.resultCoin.text = "You get $coin coins"



    }
}